### PR TITLE
Removed null usage in Repeated ctor (#888)

### PR DESCRIPTION
**Pull request by Iprogrammerr**
Solving #888 puzzle. According to recommendations by ARC (#918 ([comment](https://github.com/yegor256/cactoos/pull/918#issuecomment-402320948))), I simply removed the constructor using null and taking a Proc.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1009](https://github.com/yegor256/cactoos/issues/1009) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1009](https://github.com/yegor256/cactoos/issues/1009) is assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/f5f6466d-4ddc-4a12-b38e-e8600e25483e); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @llorllale[/z](https://www.0crat.com/u/llorllale) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by Iprogrammerr** [comment]
@victornoel Ping.

File: src/main/java/org/cactoos/func/CallableOf.java [hunk 1]
```diff
@@ -43,6 +43,9 @@
  * @param <X> Type of input
  * @param <T> Type of output
  * @since 0.12
+ * @todo #861:30min Avoid usage of null value in ctor(Func) which is
```
Comment (victornoel) [inline src/main/java/org/cactoos/func/CallableOf.java]: @Iprogrammerr please refer to the issue you are solving here

**Comment by victornoel** [comment]
@llorllale good to merge

**Comment by llorllale** [comment]
@rultor merge

**Comment by 0crat** [comment]
Code review was too long (8 days), architects (@llorllale) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by 0crat** [comment]
The job [#1009](https://github.com/yegor256/cactoos/issues/1009) is now out of scope

**Comment by 0crat** [comment]
Order was finished: +15 point(s) just awarded to @victornoel[/z](https://www.0crat.com/u/victornoel)

**Comment by 0crat** [comment]
Job `gh:yegor256/cactoos#1009` doesn't exist in WBS, can't create order

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Pull request merged**